scripts/licenses.ml

```diff
@@ -601,7 +601,7 @@ Copyright (c) 2009-2018 <the authors, see individual headers on files>
 
     |} ^ isc
   }
-  | "functoria-runtime.3.0.3" -> {
+  | "functoria-runtime.3.0.3" | "functoria-runtime.3.1.2" -> {

```

Comment:
(curious)
How do you notice the licenses need to be updated? How do you do the update?

Reply:
It's not a great system. The appveyor build fails and gives me a list, mostly consisting of new minor versions. I manually take a look to verify that they haven't changed. 99.99% of the time nothing changes so it's a bit tedious. It would be nice to automate it a bit more. In my ideal world I would just review any changed license text.

Maybe if I had a tool which could go from github tag link -> license data I could use it to update a checked-in cache of licenses.

Reply:
> Maybe if I had a tool which could go from github tag link -> license data I could use it to update a checked-in cache of licenses.

Oh, I do have that elsewhere!
I can extract a command from it, what would you need? Would that work for instance:
```
./pinata.sh license info https://github.com/bla/blo
```
?